src/content/energy-consumption/index.md

```diff
@@ -4,66 +4,66 @@ description: The basic information you need to understand Ethereum's energy cons
 lang: en
 ---
 
-# Ethereum energy consumption {#introduction}
+# Ethereum's energy expenditure {#proof-of-stake-energy}
 
-Ethereum is a green blockchain. It uses a [proof-of-stake](/developers/docs/consensus-mechanisms/pos) consensus mechanism that can be run on low-powered devices and does not require heavy computation to participate. Ethereum's proof-of-stake mechanism secures the network by using staked ETH rather than expended energy, like in [proof-of-work](/developers/docs/consensus-mechanisms/pow). The switch to proof-of-stake means the energy expended by the Ethereum network is relatively small - on the order of 0.01 TWh/yr.
+Ethereum is a green blockchain. It uses a [proof-of-stake](/developers/docs/consensus-mechanisms/pos) consensus mechanism that uses ETH rather than [energy](/developers/docs/consensus-mechanisms/pow) to secure the network. Ethereum's proof-of-stake mechanism uses just [~0.0026 TWh/yr](https://carbon-ratings.com/eth-report-2022) across the entire global network.
```

Comment:
```suggestion
Ethereum is a green blockchain. It uses a [proof-of-stake](/developers/docs/consensus-mechanisms/pos) consensus mechanism, using ETH instead of [energy to secure the network](/developers/docs/consensus-mechanisms/pow). Ethereum's proof-of-stake mechanism only uses [~0.0026 TWh/yr](https://carbon-ratings.com/eth-report-2022) across the entire global network.
```